tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -377,6 +379,33 @@ def test_chunking_fast(self):
         self.assertEqual(output, [{"text": ANY(str)}])
         self.assertEqual(output[0]["text"][:6], "ZBT ZC")
 
+    @require_torch
+    def test_chunking_fast_for_mctct(self):
+        model = MCTCTForCTC.from_pretrained("speechbrain/m-ctc-t-large")
+        processor = MCTCTProcessor.from_pretrained("speechbrain/m-ctc-t-large")
+        speech_recognizer = AutomaticSpeechRecognitionPipeline(
+            feature_extractor=processor.feature_extractor,
+            tokenizer=processor.tokenizer,
+            model=model,
+            chunk_length_s=10.0,
+            framework="pt",
+        )
```

Comment:
On a side note, I wasn't able to load this model with `pipeline` directly because there is a typo in one of the model artifacts: https://huggingface.co/speechbrain/m-ctc-t-large/blob/main/preprocessor_config.json

(`MCTCFeatureExtractor` instead of `MCTCTFeatureExtractor` (missing the `T`).)

Reply:
Thanks for the info - updating!